documentation/book/proc-deploying-example-clients.adoc

```diff
@@ -1,26 +1,29 @@
 // Module included in the following assemblies:
 //
-// assembly-hello-world.adoc
+// getting-started.adoc
 
 [id='deploying-example-clients-{context}']
-== Deploying example clients
+= Deploying example clients
+
+This example provides instructions for deploying a client to communicate with the Kafka cluster.
 
 .Prerequisites
 * An existing Kafka cluster for the client to connect to.
 
 .Procedure
 
-You can deploy the client in a single command.
-This is an example command you should substitute _test_ for your topic and _my-cluster-kafka-bootstrap:9092_ by your server address for both producer and consumer.
-
 . Deploy the producer
 +
-`oc run kafka-producer -ti --image={DockerKafka} --restart=Never \-- bin/kafka-console-producer.sh --broker-list _my-cluster-kafka-bootstrap:9092_ --topic _test_`
+[source,subs="+quotes,attributes"]
+oc run kafka-producer -ti --image={DockerKafka} --restart=Never \-- bin/kafka-console-producer.sh --broker-list __<my-producer>__:__<my-port>__ --topic _<my-topic>_
```

Comment:
Why there is a "my-producer" where we should have the bootstrap service name?

Reply:
I was trying to make it styled like the replaceable.  Using that style, what should it say? 

I can change:

__<my-producer>__

TO

__<my-cluster-kafka-bootstrap>__

Does that make sense?

Reply:
Paolo is right. The bootstrap service should be ``_<cluster-name>-kafka-bootstrap:9092_`.

Reply:
Like this?

`__<my-producer>-kafka-bootstrap__:9092`

Or:

`__<my-cluster-kafka-bootstrap>__:9092`

Reply:
I think I've got it 

`__<my-cluster>__-kafka-bootstrap:9092`

Reply:
Yes, the last one is the right one

`__<my-cluster>__-kafka-bootstrap:9092`